### DISCUSSION TITLE
do you need to use a special checkpoint to get inpainting to work?

### DISCUSSION THREAD

**Discussion by joshgura** [Acly/krita-ai-diffusion#1114] [Q&A]
brand new to krita ai diffusion. do you need to use a special checkpoint in order to get inpaint to work?

I'm getting stuck for example with "refine" as my only option when i make a selection on an already generated layer. do i need to switch modes to inpaint? fill is just creating a new layer with a white background and a contextless piece of ai art that has nothing to do with the layer i selected on.  

i'm looking forward to learning the tool, looks great.

long story short, infill is ignoring context, the generated layer has a white background that is opaque and completely replaces the original image.

**Comment by Acly** [comment]
No special checkpoint or mode.

1. Load an image
2. Make a selection using one of Krita's selection tools
3. Press Fill button

It becomes refine for less than 100% strength. 

Also if you inpaint previous generated image, make sure to apply it first.
